dracut-initramfs-restore.sh

```diff
@@ -6,20 +6,33 @@ set -e
 [ -e /run/initramfs/bin/sh ] && exit 0
 [ -e /run/initramfs/.need_shutdown ] || exit 0
 
-KERNEL_VERSION="$(uname -r)"
+[[ $KERNEL_VERSION ]] || KERNEL_VERSION="$(uname -r)"
 
 [[ $dracutbasedir ]] || dracutbasedir=/usr/lib/dracut
-SKIP="$dracutbasedir/skipcpio"
+
+if [[ -d "$dracutbasedir/skipcpio" ]]; then
+    SKIP="$dracutbasedir/skipcpio/skipcpio"
+    else
+    SKIP="$dracutbasedir/skipcpio"
+fi
+
```

Comment:
These are unrelated changes... please remove and open a new PR for them

Reply:
Removed